Makefile

```diff
@@ -72,7 +72,7 @@ tools: | $(TARGET_DIRS)
 
 progs: tools
 	git submodule update --init --recursive $(BP_COMMON_DIR)/test
-	$(MAKE) -C $(BP_COMMON_DIR)/test perch bp_tests riscv_tests beebs coremark bootrom
```

Comment:
Hm, we'll still want to provide a default bootrom to users.  Would it be possible to have a default bootrom that we relink with different ucode as part of the simulation flow?

As an aside, where does the bootrom get built in this alternate flow?

Reply:
In this flow, the bootrom gets generated whenever the user executes a program in simulation. 

We could have a default bootrom but then the way things are set up right now, a .riscv file is not re-built if it already exists in the mem folder. 

As an aside, based on earlier comments, if we are finally deciding to get rid of the domain and sac parameters from the bootrom, then maybe the bootrom can be put back into this command. The only reason it was removed was because of the -D define options being passed in during compile time (for domain and sac) to the bootrom and the fact that the CCE ucode binary was also being linked in.

Reply:
We need a default bootrom so that when people clone the repo, they can just run the examples without recompiling.  Recompiling the bootrom requires the full riscv toolchain.

>As an aside, based on earlier comments, if we are finally deciding to get rid of the domain and sac parameters from the bootrom, then maybe the bootrom can be put back into this command. The only reason it was removed was because of the -D define options being passed in during compile time (for domain and sac) to the bootrom and the fact that the CCE ucode binary was also being linked in.

Cool, that works